routes/admin.js

```diff
@@ -41,7 +41,9 @@ router
       const courses = await getCourses(id);
       return res.render("edit-university", { university, courses });
     } catch (e) {
-      return res.status(e.status).send(e);
+      return res
```

Comment:
Only change the error handling when the non error case renders a page. The rest of these changes break the existing page and don't work with the frontend code.